<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="20092" started-at="2018-10-19T00:36:51Z" finished-at="2018-10-19T00:37:11Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="20092" started-at="2018-10-19T00:36:51Z" finished-at="2018-10-19T00:37:11Z">
      <class name="com.grapeapp.qa.testcases.LoginPageTests">
        <test-method status="PASS" signature="setUpSuite()[pri:0, instance:com.grapeapp.qa.testcases.LoginPageTests@5f341870]" name="setUpSuite" is-config="true" duration-ms="437" started-at="2018-10-18T20:36:51Z" finished-at="2018-10-18T20:36:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpSuite -->
        <test-method status="PASS" signature="setUpMethod(java.lang.reflect.Method)[pri:0, instance:com.grapeapp.qa.testcases.LoginPageTests@5f341870]" name="setUpMethod" is-config="true" duration-ms="16635" started-at="2018-10-18T20:36:51Z" finished-at="2018-10-18T20:37:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.grapeapp.qa.testcases.LoginPageTests.testTitle()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpMethod -->
        <test-method status="FAIL" signature="testTitle()[pri:1, instance:com.grapeapp.qa.testcases.LoginPageTests@5f341870]" name="testTitle" duration-ms="29" started-at="2018-10-18T20:37:08Z" finished-at="2018-10-18T20:37:08Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [#1 Free CRM software in the cloud for sales and service.] but found [#1 Free CRM software in the cloud for sales and service]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [#1 Free CRM software in the cloud for sales and service.] but found [#1 Free CRM software in the cloud for sales and service]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.grapeapp.qa.testcases.LoginPageTests.testTitle(LoginPageTests.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTitle -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.grapeapp.qa.testcases.LoginPageTests@5f341870]" name="tearDownMethod" is-config="true" duration-ms="3390" started-at="2018-10-18T20:37:08Z" finished-at="2018-10-18T20:37:11Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testTitle status=FAILURE method=LoginPageTests.testTitle()[pri:1, instance:com.grapeapp.qa.testcases.LoginPageTests@5f341870] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="PASS" signature="tearDownSuite()[pri:0, instance:com.grapeapp.qa.testcases.LoginPageTests@5f341870]" name="tearDownSuite" is-config="true" duration-ms="712" started-at="2018-10-18T20:37:11Z" finished-at="2018-10-18T20:37:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownSuite -->
      </class> <!-- com.grapeapp.qa.testcases.LoginPageTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
